addons/web/static/src/views/fields/many2many_tags/many2many_tags_field.js

```diff
@@ -255,6 +255,13 @@ export class Many2ManyTagsField extends Component {
 export const many2ManyTagsField = {
     component: Many2ManyTagsField,
     displayName: _lt("Tags"),
+    supportedOptions: [
```

Comment:
There is also `placeholder` but directly as an attribute and not in the `options `. How can it be documented here ?

Reply:
Another good example for this case is the charfield with `isPassword`, `autocomplete` and `placeholder` as attrs